composer.json

```diff
@@ -39,6 +39,7 @@
         "digilive/git-changelog": "^2"
     },
     "suggest": {
+        "ext-json": "*",
```

Comment:
If in suggest, check in json renderer if extension is loaded.

Reply:
:) Good idea